client/site_config_inspect.go

```diff
@@ -76,7 +85,16 @@ func (cli *VanClient) SiteConfigInspect(ctx context.Context, input *corev1.Confi
 	if ingress, ok := siteConfig.Data["ingress"]; ok {
 		result.Spec.Ingress = ingress
 	} else {
-		result.Spec.Ingress = types.IngressLoadBalancerString
+		// check for deprecated key
+		if clusterLocal, ok := siteConfig.Data["cluster-local"]; ok {
+			if clusterLocal == "true" {
+				result.Spec.Ingress = types.IngressNoneString
+			} else {
+				result.Spec.Ingress = types.IngressLoadBalancerString
+			}
+		} else {
+			result.Spec.Ingress = types.IngressLoadBalancerString
```

Comment:
@ajssmith should we assume loadbalancer here or use `GetIngressDefault()`?

Reply:
Very good suggestion. I was keeping the previous default but now that we have GetIngressDefault() we can make a better selection.